aiorussound/connection.py

```diff
@@ -31,12 +31,17 @@ def _process_response(res: bytes) -> Optional[RussoundMessage]:
     if tag == "E":
         _LOGGER.debug("Device responded with error: %s", payload)
         raise CommandError(payload)
+
     m = RESPONSE_REGEX.match(payload.strip())
+
     if not m:
         return RussoundMessage(tag, None, None, None, None, None)
+
     p = m.groupdict()
     value = p["value"] or p["value_only"]
-    return RussoundMessage(tag, p["variable"], value, p["zone"], p["controller"], p["source"])
+    variable = p["variable"] or p["variable_only"]
```

Comment:
Added same logic that was done for value_only for variable_only as this was causing issues parsing system favorites